### ISSUE TITLE
Soporte responsive, vista horizontal/vertical

### ISSUE THREAD

**Issue by jersson** [jersson/jersson.github.io#8]
Creo que la imagen explica lo que quiero mejorar :)

![image](https://user-images.githubusercontent.com/1452411/78519460-08cb7d80-7789-11ea-9cf4-3fa48ec3ab1c.png)

Esto me ayudaría a soportar un nuevo feature que estoy diseñando, si alguien se anima, por favor me avisa, gracias!

**Comment by adagio** [comment]
¿el issue es que se monta la región de tools sobre el gráfico?

**Comment by jersson** [comment]
Sí, chartjs, parece simple pero la documentación no me ha dado pistas.
